test/functional/provider/provider_spec.lua

```diff
@@ -10,14 +10,6 @@ describe('providers', function()
     clear('--cmd', 'let &rtp = "test/functional/fixtures,".&rtp')
   end)
 
-  it('with #Call(), missing g:loaded_xx_provider', function()
-    command('set loadplugins')
-    -- Using test-fixture with broken impl:
-    -- test/functional/fixtures/autoload/provider/python.vim
```

Comment:
this test is stll valid. Change "python" to "python3" here below and move this fixture file to `python3.vim`

Reply:
Thanks.  It fixed.